### PR TITLE
2.1.0 release

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Refs: https://github.com/SemanticMediaWiki/SemanticCompoundQueries/issues/50

### CODE REVIEW

**Comment by PeterTheOne** [comment]
readme still has different requirements.. like php 5.6

**Comment by kghbln** [comment]
> readme still has different requirements.. like php 5.6

Thanks for the note. I have created the regular release issue to be able to detect such issues.

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	"require": {
 		"php": ">=7.0",
 		"composer/installers": "1.*,>=1.0.1",
-		"mediawiki/semantic-media-wiki": "~3.0"
+		"mediawiki/semantic-media-wiki": "~3.0|~3.1"
```
Comment (kghbln) [inline composer.json]: @mwjames I believe this was necessary?!

Comment (mwjames) [reply composer.json]: According to [0] ~3.0 should be equivalent to >=3.0 <4.0.0

[0] https://getcomposer.org/doc/articles/versions.md

Comment (mwjames) [reply composer.json]: You would need `|` when allowing different major versions as in case of `2.*|3.*`

Comment (kghbln) [reply composer.json]: Thanks for the info!

**Pull request merged**